dojson/utils.py

```diff
@@ -30,7 +30,7 @@ def int_with_default(value, default):
 
 
 def ignore_value(f):
-    """Remove key for None value.
+    """Remove key for None or empty value value.
```

Comment:
Have you read my proposal? There I am talking about filtering out empty values, i.e. None, empty list and empty dict (while considering False and 0 as valid values)